projects/oddz/index.js

```diff
@@ -0,0 +1,42 @@
+const sdk = require('@defillama/sdk');
+async function getTotalCollateral(pools, chain, block) {
+    const balances = {};
+    await Promise.all(pools.map(pool =>
+        sdk.api.erc20.balanceOf({
+            target: pool[1],
+            owner: pool[0],
+            chain,
+            block
+        }).then(result => sdk.util.sumSingleBalance(balances, pool[2], result.output))
+    ))
+    return balances
+}
+const bscPools = [
+    // pool, token, representation
```

Comment:
0x636f9d2Bb973D2E54d2577b9976DedFDc21E6672 sODDZ pool is considered a staking pool 
0x3c2c77353E2F6AC1578807b6b2336Bf3a3CbB014 ODDZ-BNB is a pool2 LP

Only the 0x99f29c537c70897f60c9774d3f13bd081D423467 oUSD LP would be considered in tvl